content/administer-tugboat-crew/user-admin.md

```diff
@@ -6,27 +6,38 @@ weight: 4
 
 ### User permission levels explained
 
-Tugboat has three different types of users:
+Tugboat has four different types of users:
 
+- [Owner](#owner-permissions)
 - [Admin](#admin-permissions)
 - [User](#user-permissions)
 - [Read-only](#read-only-permissions)
 
+#### Owner permissions
+
+Owner users can:
+
+- Manage billing information. This information is disabled if the company is paying with a purchase order.
+
+Owners can also do everything the [admin user](#admin-permissions) and [general user](#user-permissions) can do.
+
 #### Admin permissions
 
 Admin users can:
 
-- Manage billing information. This information is disabled if the company is paying with a purchase order.
-- Manage other users, including removing other admins, though they cannot remove themselves.
+- Add and remove users from a project, and change user permissions. Admin users can administer other admin users, though
+  they cannot remove themselves from a project.
 - Add repositories to the project.
 - Change repository settings.
 - Delete repositories.
 - Delete the entire project.
 - Rename the project.
 
+Admin can also do everything the [general user](#user-permissions) can do.
+
 #### User permissions
 
-Tugboat's generic User's permissions include:
+Tugboat's general User's permissions include:
 
 - Manage the repository configuration interface. This includes things like
```

Comment:
This needs to be moved up to the Admin section. Users can view repo settings, but can't change them.

Reply:
Added this to the Admin user section, but left a mention here that users can View repo settings.